src/main.js

```diff
@@ -10,6 +10,11 @@ var app = Elm.App.fullscreen({
   deduction: localStorage.getItem('deduction'),
 });
 
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker
+    .register('./service-worker.js')
```

Comment:
would you mind add semicolons? I don't want to start flame war about semicolons or not... but I like conventions, so we should stick to the same linting rules.